client/swarm_get_unlock_key_test.go

```diff
@@ -0,0 +1,60 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestSwarmGetUnlockKeyError(t *testing.T) {
+	client := &Client{
+		client: newMockClient(errorMock(http.StatusInternalServerError, "Server error")),
+	}
+
+	_, err := client.SwarmGetUnlockKey(context.Background())
+	testutil.ErrorContains(t, err, "Error response from daemon: Server error")
```

Comment:
It's the same as the others (for other endpoint). Making sure we return this error in case of server error. It's less *useful* than when we have more to support (like on some container endpoint, where we handle `NotFound` errors, etc..)